exercises/variable_exercise_2.py

```diff
@@ -0,0 +1,71 @@
+#integer
+x = 5
+print(x)
+
+
```

Comment:
Just an FYI, so extra empty lines are allowed in python and do sometimes add to clarity, but in general we do not want to have more than one extra empty line.  We will talk about formatting after this.  If you have time I would suggest checking out a python auto-formatter called `Black`.

Additionally, I would also install Pycharm, which is a professional IDE (use the community version, it's free).  I can help orient you on that later as well.

Reply:
OK good to know.